cocos/platform/android/java/src/org/cocos2dx/lib/Cocos2dxWebView.java

```diff
@@ -66,6 +68,13 @@ public Cocos2dxWebView(Context context, int viewTag) {
 
         this.setWebViewClient(new Cocos2dxWebViewClient());
         this.setWebChromeClient(new WebChromeClient());
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.KITKAT) {
```

Comment:
There is no compiling issues, if CppTest is run from `tests/cpp-tests/proj.android-studio` directory (Jenkins had a build break during compilation of `tests/cpp-tests/proj.android`).